data/scripts.ts

```diff
@@ -1085,8 +1079,7 @@ export const Scripts: BattleScriptsData = {
 	},
 
 	calcRecoilDamage(damageDealt, move) {
-		// @ts-ignore
-		return this.clampIntRange(Math.round(damageDealt * move.recoil[0] / move.recoil[1]), 1);
+		return this.clampIntRange(Math.round(damageDealt * move.recoil[0]! / move.recoil[1]!), 1);
```

Comment:
```suggestion
		return this.clampIntRange(Math.round(damageDealt * move.recoil![0] / move.recoil![1]), 1);
```

I _think_ this is what you want?

Reply:
Yes, sorry, I mixed it up when I tried to undo my trial `calRecoilDamage` change as mentioned in the original description.

Reply:
You don't need to apologize so much. Especially since this is a really useful change overall, and all these things are easy mistakes or nitpicks.

Reply:
I feel a strange need to apologise for apologising too much...